weathergov/frontend/assets/sass/components/daily-summary.scss

```
@@ -0,0 +1,44 @@
+@use "uswds-core" as *;
+@forward "uswds-theme";
+@forward "uswds";
+@forward "daily";
+
+.wx-daily-forecast-summary-area--other {
+    margin-bottom: units(1);
+    @include grid-col(6);
```

I can't find this in the resolved comments, so it's possible it didn't save on my first review: 

I think we need to wrap this is in a media query otherwise it creates a narrow column on mobile

```suggestion
    @include at-media(tablet) {
        @include grid-col(6);
}
```